olive/common/config_utils.py

```diff
@@ -239,16 +239,6 @@ def validate_object(v, values, field):
     return v
 
 
-def validate_resource_path(v, values, field):
```

Comment:
This is a good catch. The code inside olive.common are not expected to depend on any other olive code other than those under the 'olive.common'